buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All

```diff
@@ -291,6 +291,8 @@ try {
 
 def build(JOB_NAME, OPENJDK_REPO, OPENJDK_BRANCH, SHAS, OPENJ9_REPO, OPENJ9_BRANCH, OMR_REPO, OMR_BRANCH, SPEC, SDK_VERSION, BUILD_NODE, TEST_NODE, EXTRA_GETSOURCE_OPTIONS, EXTRA_CONFIGURE_OPTIONS, EXTRA_MAKE_OPTIONS, OPENJDK_CLONE_DIR, ADOPTOPENJDK_REPO, ADOPTOPENJDK_BRANCH, AUTOMATIC_GENERATION, CUSTOM_DESCRIPTION) {
```

Comment:
How does `build` differ from the one in pipeline-functions?

Reply:
The `build` here (context PBT-All) is a function to call the PBT-Any jobs. `build` in `pipeline-functions` is the function used to call the BT-Any jobs. Admittedly a bad choice of names. We could rename the PBTA one to be `build_pipeline`.